docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -23,28 +25,28 @@ Want to use {agent} instead? Refer to <<monitoring-with-elastic-agent>>.
 ==== Disable default collection of {ls} monitoring metrics
 
 --
-The `monitoring` setting is in the {ls} configuration file (logstash.yml), but is
-commented out: 
+Set the `monitoring.enabled` to `false` in logstash.yml to disable to default monitoring:
 
 [source,yaml]
 ----------------------------------
 monitoring.enabled: false
 ----------------------------------
 
-Remove the `#` at the beginning of the line to enable the setting.
-
 --
 
 [float]
 [[define-cluster__uuid]]
-==== Define `cluster_uuid` (optional)
-To bind the metrics of {ls} to a specific cluster, optionally define the `monitoring.cluster_uuid`
+==== Define `cluster_uuid`
+If your each pipeline contains <<plugins-outputs-elasticsearch,{es} output plugin>>,
+*Stack Monitoring* automatically figures out the `cluster_uuid` and no need to specify it.
+To bind the metrics of {ls} to a specific cluster, define the `monitoring.cluster_uuid`
 in the configuration file (logstash.yml):
```

Comment:
review note: if any of pipelines don't define `es-output`, monitoring cluster becomes an orphan, so removing the _(optional)_.